gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/spring/ResourceContextConfiguration.java

```diff
@@ -324,4 +328,14 @@ public ApiMetadataDecoderDomainService apiMetadataDecoderDomainService() {
     public ValidateApiDomainService validateApiDomainService() {
         return mock(ValidateApiDomainService.class);
     }
+
+    @Bean
```

Comment:
This should not be necessary. We should declare the in memory alternative only in https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/inmemory/spring/InMemoryConfiguration.java